src/data-fetchers/gff/gff-worker.ts

```diff
@@ -26,7 +26,7 @@ export interface EmptyTile {
 /**
  * A class to represent a GFF file. It takes care of setting up gmod/tabix.
  */
-export class GffFile {
+class GffFile {
```

Comment:
I'm guessing this is just for consistency?

Reply:
Yep! I started addressing some of the unused exports that knip identified.